clients/admin-ui/src/pages/index.tsx

```diff
@@ -1,17 +1,61 @@
-import { Heading } from "@fidesui/react";
+import { Box, Button, Flex, Heading } from "@fidesui/react";
 import type { NextPage } from "next";
+import Head from "next/head";
+import NextLink from "next/link";
 import React from "react";
 
-import Layout from "~/features/common/Layout";
+import Header from "../features/common/Header";
+// import Setup from './setup';
+import { ArrowDownLineIcon } from "../features/common/Icon";
+
+const Home: NextPage<{ session: { username: string } }> = ({ session }) => (
+  <div>
+    <Head>
+      <title>Fides Admin UI</title>
+      <meta name="description" content="Generated from FidesUI template" />
+      <link rel="icon" href="/favicon.ico" />
+    </Head>
+
+    <Header username={session && session.username} />
 
-const Home: NextPage = () => (
-  <Layout title="Home">
     <main>
-      <Heading mb={8} fontSize="2xl" fontWeight="semibold">
-        Welcome home!
-      </Heading>
+      {/* NEED TO FLAG BEFORE RELEASE
+      Show Setup when the user is an admin and there are zero systems registered in the database */}
+      {/* <Setup /> */}
+
+      <Flex
```

Comment:
Actually it looks like this might've been copy pasta'd from fidesops? Can we instead just add the changes you need to the index.tsx that has the updated navbar? I had a few outstanding PR's that I've been merging today, so maybe this got put in before all the merging, sorry about that!